CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ### Changed
 - Internal: Redux and EventEmitter are not in the global scope anymore. The `tearDown` function will only unmount the SDK.
+- Internal: Use `v2/snapshots` for multiple frame selfie captures
```

Comment:
Very Minor: I feel it's clearer to specify that `v2/snapshots` is an endpoint in documentation here